### PR TITLE
blitz_doc_update_mziabari

### PR DESCRIPTION
**Pull request by KamyarZiabari**
New branch for blitz_doc

### CODE REVIEW

File: develop-guide/writetrigger/writetrigger.rst [hunk 1]
```diff
@@ -661,17 +657,17 @@ somewhere in the output, irrelevant of the structure.
 
         # Can have as many items under include or exclude that you want
         include:
-            - key: "[version][version]"
-              value: 16.9.1
-            - key: "[version][main_mem]"
+            # Using Dq (the new dictionary querying tool), you can verify whether
```
Comment (jeaubin) [inline develop-guide/writetrigger/writetrigger.rst]: provide link to dq doc

https://pubhub.devnetcloud.com/media/genie-docs/docs/userguide/utils/index.html#dq

File: develop-guide/writetrigger/writetrigger.rst [hunk 2]
```diff
@@ -844,70 +921,12 @@ devices in parallel.
...
-              arguments:
-                interface: GigabitEthernet1
-          - configure:
-              device: PE1
-              command: |
-                router bgp '%VARIABLES{output}'
-
-        ...
-
+Please note that you cannot save a variable in parallel and immideately use it in another action
```
Comment (jeaubin) [inline develop-guide/writetrigger/writetrigger.rst]: immideately    typo

File: develop-guide/writetrigger/writetrigger.rst (hunk 1 above)
Comment (KamyarZiabari) [reply develop-guide/writetrigger/writetrigger.rst]: I did add the link, but this is in the code block and it looks weird to me. I actually below mentioned Dq and gave link to it.

File: develop-guide/writetrigger/writetrigger.rst [hunk 3]
```diff
@@ -752,20 +762,87 @@ similar to api action and parse action.
...
+              arguments:
+                interface: GigabitEthernet1
+          - configure:
+              device: PE1
+              command: |
+                router bgp '%VARIABLES{api_output}'
+        ...
+
+Both filter and include/exclude features are using a dictionary querying tool called `Dq
+<https://pubhub.devnetcloud.com/media/genie-docs/docs/userguide/utils/index.html#dq>`_.
```
Comment (KamyarZiabari) [inline develop-guide/writetrigger/writetrigger.rst]: Here I gave the link

File: develop-guide/writetrigger/writetrigger.rst (hunk 2 above)
Comment (KamyarZiabari) [reply develop-guide/writetrigger/writetrigger.rst]: Done

**Pull request merged**